Makefile

```diff
@@ -8,20 +8,25 @@
 
 SIL ?= @
 MAKEFLAGS += --no-print-directory
+VERSION="1.21 rev: "`git rev-parse --short HEAD`
 
 KERNEL=1
 
 include ../phoenix-rtos-build/Makefile.common
 include ../phoenix-rtos-build/Makefile.$(TARGET_SUFF)
 
+CFLAGS += $(BOARD_CONFIG) -DVERSION=\"$(VERSION)\"
+CFLAGS += -I../plo
+CFLAGS += -DPLO_CONFIG=\"hal/$(TARGET_SUFF)/$(TARGET_SUBFAMILY)/config.h\" -DARCH_TYPES=\"hal/$(TARGET_SUFF)/$(TARGET_SUBFAMILY)/types.h\"
```

Comment:
instead of `-DPLO_CONFING` You should add `-Ihal/$(TARGET_SUFF)/$(TARGET_SUBFAMILY)` and include `config.h` in code directly